src/open_inwoner/qmatic/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+from zgw_consumers.constants import APITypes
+
+
+class QmaticConfigManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().select_related("service")
+
+
+class QmaticConfig(SingletonModel):
+    """
+    Global configuration and defaults
+    """
+
+    service = models.OneToOneField(
+        "zgw_consumers.Service",
+        verbose_name=_("Calendar API"),
+        on_delete=models.PROTECT,
+        limit_choices_to={"api_type": APITypes.orc},
+        related_name="+",
```

Comment:
Not very descriptive name. Intentional?

Reply:
Setting the related_name to `"+"` makes sure that `Service` doesn't have a backwards relation to `QmaticConfig` (which I think is right in this case, because trying to get a `SingletonModel` from a `Service` instance isn't right): https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.ForeignKey.related_name